### PR TITLE
Ja.w 8767284/uppercase methods

### PR DESCRIPTION
**Pull request by monkey-jeff**
This fixes a bug where our servers would reject methods if the verb wasn't in all caps.  

I am not sure how to write a specific test for this as nock doesn't care about verb case in their matcher.

### CODE REVIEW

**Review by dbarbato-sf — APPROVED** [review]

**Comment by kalagarraj** [comment]
> This fixes a bug where our servers would reject methods if the verb wasn't in all caps.
> 
> I am not sure how to write a specific test for this as nock doesn't care about verb case in their matcher.

Approving, i think nock has options to pass in a custom matcher..

**Review by kalagarraj — APPROVED** [review]

File: scripts/templateHelpers.ts [hunk 1]
```diff
@@ -126,3 +126,12 @@ export const isCommonPathParameter = (property: string): boolean => (property
...
   : false);
+
+/**
+ * Forces the input to all caps.
+ *
+ * @param input - The string you want to change to all caps
+ *
+ * @returns string in all caps
+ */
+export const loud = (input: string): string => String(input).toUpperCase();
```
Comment (shirishmaheshwari) [inline scripts/templateHelpers.ts:137]: Any reason for calling the variable 'loud'?

**Review by shirishmaheshwari — APPROVED** [review]

File: scripts/templateHelpers.ts (hunk 1 above)
Comment (shirishmaheshwari) [reply scripts/templateHelpers.ts:137]: I see. all caps -> loud :)

**Review by bredmond-sf — APPROVED** [review]

**Comment by monkey-jeff** [comment]
> > This fixes a bug where our servers would reject methods if the verb wasn't in all caps.
> > I am not sure how to write a specific test for this as nock doesn't care about verb case in their matcher.
> 
> Approving, i think nock has options to pass in a custom matcher..

I tried that...but the method always got to the matcher in all caps so I couldn't make a negative test for it.

**Pull request merged**